charts/node-analyzer/values.yaml

```
@@ -318,10 +318,11 @@ nodeAnalyzer:
     # additionalDirsToScan is a optional comma-separated list of directories that
     # should be analyzer in addition to default ones.
     # additionalDirsToScan: "/foo/bar/baz,/my/other/folder"
+    probesPort: 7001
```

please @dippie8 add a comment here (basically the README's one)
```suggestion

    // probesPort ...
    probesPort: 7001
```